Platform/OpenCanopy/OcBootstrap.c

```diff
@@ -248,13 +249,6 @@ OcShowPasswordByOc (
 
   mDrawContext.TimeOutSeconds = 0;
 
-  //
-  // Do not play intro animation for blind.
-  //
-  if (Context->PickerAudioAssist) {
-    mGuiContext.DoneIntroAnimation = TRUE;
-  }
-
```

Comment:
mGuiContext.DoneIntroAnimation was never used or checked in the GUI password display (the fade in just happens always).